### ISSUE TITLE
Windows Source Build Seems to Currently Fail

### ISSUE THREAD

**Issue by Ryonez** [armory3d/armortools#1584]
**ArmorPaint version:** [0326daa](https://github.com/armory3d/armortools/commit/0326daa703742e1b4f6b07d2a750dafce3bd1978), [041429a](https://github.com/armory3d/armortools/commit/041429a6ad953828a82ca39d4306feb029a44bc7)]

**OS/device including version:** Windows 11 Pro 22H2 Build 22621.1848

**Issue description:** An exception is thrown when debugging the release x64 build. (Running standalone causes a CTD)
![image](https://github.com/armory3d/armortools/assets/16157130/03eeb177-5990-45f9-ae27-4f9d716d25cd)

**Steps to reproduce:**
* Follow the Windows guide in the readme
* Make sure that the Release configuration for x64 is selected.
* Select `Local Windows Debugger`

It is possible to "run" without the debugger, but the application crashes to desktop after opening.

**Comment by luboslenco** [comment]
Thanks for report. Seems to work normally now, please try updating to the latest git revision.

**Comment by Ryonez** [comment]
Nope, still doesn't work for https://github.com/armory3d/armortools/commit/52f4b29ee96de34f9b2726818762bb904e3daca6

And please don't close issues before receiving feedback from the one having the trouble. Works for you doesn't mean works for me.

**Comment by Ryonez** [comment]
@luboslenco

**Comment by luboslenco** [comment]
> And please don't close issues before receiving feedback from the one having the trouble. Works for you doesn't mean works for me.

Don't want to sound rude so please don't take it like that - but the main goal of the tracker is to improve the program for the majority of people, not provide individual build support. I try to provide support on build issues as well, but I am not sure what is going wrong there, sorry. :( If you ever figure it out then additional comments are welcome.

**Comment by Ryonez** [comment]
>to improve the program for the majority of people, not provide individual build support

Given issue are started by individual's, should they not even bother then?
It's kinda frustrating to have this issue and just had it waved away. How many people here are compiling for windows themselves?

I opened this a while ago. Since then I've reinstalled the os. After you said it worked for you, there was a fresh install of Visual Studio Code when I went to try again.
I also know nothing about your environment, like are you using the same OS version? Did you try the versions I mentioned had an issue to confirm you had the issue in them, or was it just, update -> build -> looks good?
